src/olympia/editors/tests/test_views.py

```diff
@@ -1980,12 +1858,13 @@ def test_breadcrumbs(self):
         self._test_breadcrumbs(expected)
 
     def test_breadcrumbs_unlisted_addons(self):
-        self.addon.update(is_listed=False)
+        self.addon.update(is_listed=False, status=amo.STATUS_PUBLIC)
```

Comment:
Oh, I guess at the moment all unlisted add-ons will look like that.  But when there are mixed listed/unlisted add-ons?  Hmm.